### PR TITLE
PR:Button Component

### PR DESCRIPTION
**Pull request by Jayesh3636**
- Added Button Component

  - Standard Buttons
  - Outline Buttons
  - Button with Icons
  - Button with Links
  - Floating Action Buttons
  
  
  Kindly review and let me know the changes! TIA!

https://user-images.githubusercontent.com/91378157/153679909-734a85be-f17d-42bf-9aae-ecbd3c3ea168.mp4

### CODE REVIEW

**Review by pavan6115 — APPROVED** [review]
Looks good 👍

File: Components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+    color: var(--white);
+}
+
+/* link text primary */
+.link-text-primary {
+    text-decoration: underline;
+    text-underline-position: under;
+    text-decoration-color: var(--primary-color);
+    padding: 0;
+    color: #5b5b5b;
```
Comment (shubhmehra) [inline Components/button/button.css]: you can use color from :root to make this consistent

File: Components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+    color: var(--white);
+}
+
+/* link text primary */
+.link-text-secondary {
+    text-decoration: underline;
+    text-underline-position: under;
+    text-decoration-color: var(--secondary-color);
+    padding: 0;
+    color: #5b5b5b;
```
Comment (shubhmehra) [inline Components/button/button.css]: same as above!

File: Components/button/button.html [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+                    <div class="nav__logo">
+                        <img src="../../images/DARKBUCKET-title.svg" class="nav__logo">
+                    </div>
+                </a>
+            </div>
+            <div class="nav__right">
+                <div class="social-media-list">
+                    <ul class="social-media-list">
+                        <li class="social-media-items"><a href="#"><img class="icon-height"
+                                    src="../../images/Linkedin.png" alt="Linkedin"></a></li>
```
Comment (shubhmehra) [inline Components/button/button.html:32]: As from your folder structure I think you can directly go like this --> /images/linkdin.png
same you can check in others.

File: Components/button/button.html [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+                                </button>
+                            </div>
+                        </div>
+
+                        <div class="button__floating-iframe">
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=paraiso-dark&wt=none&l=css&width=1137&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=dm&fs=16.5px&lh=154%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520btn-float%2520float-up%2522%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522material-icons%2522%253Earrow_upward%253C%252Fspan%253E%250A%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-float%2522%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522material-icons%2522%253Eadd%253C%252Fspan%253E%250A%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-float%2522%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522material-icons%2522%253Earrow_downward%253C%252Fspan%253E%250A%253C%252Fbutton%253E"
+                                style="width: 1137px; height: 303px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (shubhmehra) [inline Components/button/button.html:215]: carbon in line styling can be optimised width: 100% according to your container can make it responsive.
or you can add common class in your main css for this too.

File: Components/button/button.css (hunk 2 above)
Comment (Jayesh3636) [reply Components/button/button.css]: done

File: Components/button/button.html (hunk 3 above)
Comment (Jayesh3636) [reply Components/button/button.html:32]: tried resolving this with your method but it wasn't working for me as I have my file structure like this!

image
all components are in components and images are in images according to component folder
image




In Components it looks like this  each component is in new folder
image


and in button

image



so to use any image i have to come two folders back to access images!
image


thats why using src="../../images/Linkedin.png"

File: Components/button/button.css (hunk 1 above)
Comment (Jayesh3636) [reply Components/button/button.css]: done~~!

File: Components/button/button.html (hunk 3 above)
Comment (shubhmehra) [reply Components/button/button.html:32]: cool then go ahead with current structure if that is the case!

**Pull request merged**